### PR TITLE
ci: support sub-accounts

### PR DESCRIPTION
**Pull request by luislhl**
### Motivation

We need to deploy to several new environments

### Acceptance Criteria

- The deploy script should support deploying to the new environments.

### Checklist
- [ ] If you are requesting a merge into `master`, confirm this code is production-ready and can be included in future releases as soon as it gets merged
- [ ] Make sure either the unit tests and/or the QA tests are capable of testing the new features
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by andreabadesso — APPROVED** [review]

File: .codebuild/build.sh [hunk 1]
```diff
@@ -9,94 +9,219 @@ send_slack_message() {
...
+        make build-daemon;
+
+        # --- Testnet ---
+        # Gets all env vars with `testnet_` prefix and re-exports them without the prefix
+        for var in "${!testnet_@}"; do
+            export ${var#testnet_}="${!var}"
+        done
+
+        make migrate;
+        make deploy-lambdas-testnet;
```
Comment (pedroferreira1) [inline .codebuild/build.sh:78]: Why did you remove `make push-daemon;` after this? Do you need to run it only once for both envs?

File: .codebuild/build.sh [hunk 2]
```diff
@@ -9,94 +9,219 @@ send_slack_message() {
...
+            unset ${var#testnet_}
+        done
+
+        # --- Mainnet ---
+        # Gets all env vars with `mainnet_` prefix and re-exports them without the prefix
+        for var in "${!mainnet_@}"; do
+            export ${var#mainnet_}="${!var}"
+        done
+        make migrate;
+        make deploy-lambdas-mainnet;
```
Comment (pedroferreira1) [inline .codebuild/build.sh:91]: Why did you remove `make build-daemon;`? Do you need to run only once for both envs?

File: .codebuild/build.sh (hunk 1 above)
Comment (luislhl) [reply .codebuild/build.sh:78]: Yes, both Testnet and Mainnet use the same Docker images currently. Building them twice as we did before as unnecessary.

File: .codebuild/build.sh (hunk 2 above)
Comment (luislhl) [reply .codebuild/build.sh:91]: Yes, both Testnet and Mainnet use the same Docker images currently. Pushing them twice as we did before as unnecessary.

**Review by pedroferreira1 — APPROVED** [review]
